students/noahjones

```diff
@@ -0,0 +1,351 @@
+        }
+        
+        .eyes {
+            position: relative;
+            top: 58px;
+            left: 40px;
+            display: flex;
+        }
+        
+        .eyes .eye {
+            position: relative;
+            width: 80px;
+            height: 80px;
+            display: block;
+            background: #fff;
+            margin: 0 15px;
+            border-radius: 50%;
+        }
+        
+        .eyes .eye::before {
+            content: '';
+            position: absolute;
+            top: 50%;
+            left: 25px;
+            transform: translate(-50%, -50%);
+            width: 40px;
+            height: 40px;
+            background: #333;
+            border-radius: 50%;
+        }
+    </style>
+
+    <title>NJ Jones Tech Profile Page</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <div class="m">
+            J
+        </div>
+        <div class="e">
+            O
+        </div>
+        <div class="g">
+            N
+        </div>
+        <div class="a">
+            E
+        </div>
+        <div class="n">
+            S
+        </div>
+        <div class="ex">
+            !
+        </div>
+
+    </div>
+
+    <body>
+        <div class="face">
+            <div class="eyes">
+                <div class="eye"></div>
+                <div class="eye"></div>
+            </div>
+        </div>
+    </body>
+    <script>
+        document.querySelector('body').addEventListener('mousemove', eyeball);
+
+        function eyeball() {
+            var eye = document.querySelectorAll('.eye');
+            eye.forEach(function(eye) {
+                let x = (eye.getBoundingClientRect().left) + (eye.clientWidth / 2);
+                let y = (eye.getBoundingClientRect().top) + (eye.clientHeight / 2);
+                let radian = Math.atan2(event.pageX - x, event.pageY - y);
+                let rot = (radian * (180 / Math.PI) * -1) + 270;
+                eye.style.transform = "rotate(" + rot + "deg)"
+            })
+        }
+    </script>
+    <center>
+        <button><a href="http://wwbot-wedev.surge.sh">HOME</a>
+            </button>
+        <button><a href="http://www.noahjones.net">MY RESUME</a>
+            </button>
+        <button><a href="http://www.njartworks.com">MY ART</a>
+            </button>
+        <div>
+            <p>
+                <h1>Thank you, for visiting my profile page.</h1>
+
+            </p>
+        </div>
+        <br>
+
+        <div class="box">
+            <border color="#000000">
+                </font><img src="profilepic.jpg" , width="10%"></border>
+        </div>
+        <font color="#ffffff">
+            <p style="color:khaki;">My name is Noah Jones and I am a Elementary School computer teacher. I have 10 years of experience working as a teacher.
+                <BR>I also work as a digital media artist and designer, I have been working in the area of visual arts and graphic design for over 20 years.</p>
```

Comment:
💭 Instead of using a `<br>` tag, we can create a new `<p>` tag which will give you the same result. 

It's also customary to use all lowercase tags in HTML.

Reply:
Made this edit